system/jlib/jsocket.cpp

```diff
@@ -5151,30 +5213,41 @@ ISocketSelectHandler *createSocketSelectHandler(const char *trc)
         if (epoll_method == EPOLL_INIT)
         {
             if (queryEnvironmentConf().getPropBool("use_epoll", true))
+            {
                 epoll_method = EPOLL_ENABLED;
+                epoll_hdlPerThrd = queryEnvironmentConf().getPropInt("epoll_hdlperthrd", -1);
```

Comment:
You should get a warning here (on some compilers at least) re. implicit conversion from int to unsigned, so should add a cast to unsigned.